modules/rosa-delete-users.adoc

```diff
@@ -0,0 +1,20 @@
+
+// Module included in the following assemblies:
+//
+// getting_started_rosa/rosa-creating-cluster.adoc
+
+
+[id="rosa-delete-users"]
+= Removing a user through the OCM console
+To remove full access of users so that they cannot access the cluster, you must remove the users through the OpenShift Cluster Manager (OCM) console in addition to revoking their `dedicated-admin` or `cluster-admin` privileges.
```

Comment:
After revoking by `rosa revoke user` command, the user should delete from dedicated-admin/cluster-admin and the users info on OCM console will also be deleted.

Reply:
This is not entirely correct.  Removing access to the cluster is done via the IdP and not OCM.  OCM UI or the ROSA CLI can only be used to remove the elevated access.

Reply:
Updated the intro.